README.md

```diff
@@ -16,68 +16,75 @@ To get started with the demo app, please ensure Flutter is installed on your sys
 4. Update the package name from `com.applovin.enterprise.apps.demoapp` to one that matches your ad units. Be sure to do this for every package name reference in the demo app. 
 
 ### Android
-#### 1. Adding Adapters to `build.gradle`:
+#### 1. Updating the package name in `app/build.gradle`:
 - Navigate to your Flutter project directory in your file explorer or terminal. 
 - Within the project directory, navigate to `android/app/` to find the `build.gradle` file. 
 - Open `build.gradle` with a text editor or an IDE. 
-- Add the adapters you need under dependencies, as the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#android) specifies. It will look something like this:
+- Update `applicationId` with your package name.
+````
+android {
+    ...
+    defaultConfig {
+        applicationId "your_package_name"
+        ...
+    }
+````
+
+#### 2. Adding adapters to `app/build.gradle`:
+- Add the necessary adapters for the mediated ad networks you plan to integrate, as specified in the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#android). It will look something like this:
 ```
 dependencies {
     // Other dependencies...
     implementation 'com.example.adapter:version'
 }
 ```
+> [!CAUTION]
+> Do not add the underlying AppLovin SDK to your dependencies. The AppLovin MAX Flutter plugin
+> already specifies the certified SDK version corresponding to the plugin. Manually adding it may
```

Comment:
[I recommend adding the subject:](https://docs.google.com/document/d/1mvaE1khnBVsZA30Mau_S4bfjbqyB9uxoPXEDN45hopY/edit#heading=h.w4angfuaeym9)
"**If you also manually add the dependency**, this may break your build and cause unpredictable results."